app/views/event_mailer/transparency_mode_disabled.html.erb

```diff
@@ -0,0 +1,16 @@
+<p><%= @emails.length > 1 ? "Hi all," : "Hi there," %></p>
+
+<p>
+  A manager on your team has disabled transparency mode for <%= @event.name %>.
+  This means your organization is now private to unauthenticated users on HCB.
+  Any published announcements are still visible to the public.
```

Comment:
Conditionally render

> Any published announcements are still visible to the public.

depends on whether they have published announcements or not